server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -51,14 +56,17 @@ abstract class ProgramFormBuilder extends BaseHtmlView {
   private final SettingsManifest settingsManifest;
   private final String baseUrl;
   private final AccountRepository accountRepository;
+  private final CategoryRepository categoryRepository;
```

Comment:
(question, blocking) The image above shows the category checkboxes sorted, but I didn't see any code that is sorting the list. Is this being sorted case-insensitively outside this PR? If not, can you put a sort on it.

Reply:
Added sorting in the `CategoryRepository.listCategories`.  They were only sorted previously because of the order of insertion into the db.